Assets/MixedRealityToolkit.SDK/Inspectors/Utilities/Solvers/AttachToControllerInspector.cs

```diff
@@ -6,7 +6,7 @@
 using UnityEditor;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.UI.Inspectors.Utilities.Solvers
+namespace Microsoft.MixedReality.Toolkit.UI.Editor.Solvers
```

Comment:
I know that this was already discussed on the chat, but I do wanna call out that some other solvers are in this namespace:

namespace Microsoft.MixedReality.Toolkit.Utilities.Solvers

[https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/ConstantViewSize.cs](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/ConstantViewSize.cs)
[https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/InBetween.cs](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/InBetween.cs)
[https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Overlap.cs](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Overlap.cs)

I don't really have a strong opinion between any of these so the power is yours.

Reply:
I will revisit all solvers namespace tomorrow.

---
In reply to: [276877813](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/4019#discussion_r276877813) [](ancestors = 276877813)

Reply:
Also, facepalm on my part, I totally missed the inspector word =/

---
In reply to: [276878366](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/4019#discussion_r276878366) [](ancestors = 276878366,276877813)